qds_sdk/cloud/cloud.py

```diff
@@ -0,0 +1,16 @@
+from qds_sdk.qubole import Qubole
+
+class Cloud:
+
+    @staticmethod
+    def get_cloud():
+        cloud = Qubole.cloud
+        if cloud == "azure":
+            import qds_sdk.cloud.azure_cloud
+            return qds_sdk.cloud.azure_cloud.AzureCloud()
+        elif cloud == "oracle_bmc":
+            import qds_sdk.cloud.oracle_bmc_cloud
+            return qds_sdk.cloud.oracle_bmc_cloud.OracleBmcCloud()
+        else:
+            import qds_sdk.cloud.aws_cloud
+            return qds_sdk.cloud.aws_cloud.AwsCloud()
```

Comment:
We should define functions `set_cloud_config_from_arguments` and `create_parser ` in cloud.py so that any class which extends this class in future knows that it has to implement these functions.